vortex-api/src/main/java/mil/army/usace/hec/vortex/io/NetcdfDataReader.java

```diff
@@ -342,8 +356,13 @@ private List<VortexData> getData(GridDataset dataset, String variable) {
         return grids;
     }
 
-    private static String getWkt(Projection projection) {
-        return WktFactory.createWkt((ProjectionImpl) projection);
+    private static String getWkt(GridCoordSystem gridCoordSystem) {
```

Comment:
This function would retrieve the `crs` attribute from the `nc` file if it exists. If not, it would use the `Projection` attribute -- which would miss some coordinate system data.

Reply:
I think we should look at this together.